users/templates/partials/admin_user_update_form.html

```diff
@@ -0,0 +1,32 @@
+{% load crispy_forms_tags %}
+{% load i18n %}
+
+<div class="row form-row">
+    <div class="form-group col-md-6 mb-0">
+        {{ form.first_name|as_crispy_field }}
+    </div>
+    <div class="form-group col-md-6 mb-0">
+        {{ form.last_name|as_crispy_field }}
+    </div>
+</div>
+<div class="row form-row">
+    <div class="form-group col-md-6 mb-0">
+        {{ form.email|as_crispy_field }}
+    </div>
+    <div class="form-group col-md-6 mb-0">
+        {{ form.user_type|as_crispy_field }}
+    </div>
+</div>
+<div class="form-group row">
+    <div class="form-group col-md-4 mb-0">
+        <input type="button" id="opener_user_account_update" class="btn btn-success" name="update"
+           value="{% trans 'Update' %}">
+    </div>
+    <div class="form-group col-md-4 mb-0">
+        <input type="button" class="btn btn-secondary" name="discard" value="{% trans 'Discard' %}"
```

Comment:
Due to different interpretations of `btn-secondary` class by browsers, you have to write your own class in css.

Safari Mac OS X version 13.0 (15608.1.37)
![image](https://user-images.githubusercontent.com/37847362/63517911-2eed2780-c4f0-11e9-9ad8-bdc322f51229.png)

Opera Mac OS X version 62.0.3331.119
![image](https://user-images.githubusercontent.com/37847362/63517620-95257a80-c4ef-11e9-8f13-d828b9d9ae01.png)

Chrome Mac OS X version 76.0.3809.100 (official) (64-bit)
![image](https://user-images.githubusercontent.com/37847362/63517980-52b06d80-c4f0-11e9-9d4e-df65e013458b.png)